### DISCUSSION TITLE
Integration with To-do Lists

### DISCUSSION THREAD

**Discussion by bigbeefyangus** [VolantisDev/ha-chore-helper#43] [Ideas]
Hello! I just started using this integration and I like the way you've handled creating recurring chores. With the new To-do list integration from 2023.11, I'm curious if there are any plans to be able to integrate chores into the to-do list built into HA as I feel this is a better implementation that the standard "recurring date" function in to-do. I do like the calendar included with this helper, but I find my calendar gets cluttered very quick.

I'm assuming a workaround would be to create an automation to update a to-do list item, but I'm assuming this would have to be done for each chore? (sidenote: I'm also interested to know how users are displaying chores on their dashboards currently).

Keep up the awesome work!

**Comment by flemmingss** [comment]
Agree 100%
